controllers/openstackdataplanenodeset_controller.go

```diff
@@ -44,47 +45,14 @@ import (
-	// LogrotateDefaultImage -
-	LogrotateDefaultImage = "quay.io/podified-antelope-centos9/openstack-cron:current-podified"
-	// NeutronMetadataAgentDefaultImage -
-	NeutronMetadataAgentDefaultImage = "quay.io/podified-antelope-centos9/openstack-neutron-metadata-agent-ovn:current-podified"
-	// NovaComputeDefaultImage -
-	NovaComputeDefaultImage = "quay.io/podified-antelope-centos9/openstack-nova-compute:current-podified"
-	// OvnControllerAgentDefaultImage -
-	OvnControllerAgentDefaultImage = "quay.io/podified-antelope-centos9/openstack-ovn-controller:current-podified"
-	// OvnBgpAgentDefaultImage -
-	OvnBgpAgentDefaultImage = "quay.io/podified-antelope-centos9/openstack-ovn-bgp-agent:current-podified"
-	// TelemetryCeilometerComputeDefaultImage
-	TelemetryCeilometerComputeDefaultImage = "quay.io/podified-antelope-centos9/openstack-ceilometer-compute:current-podified"
-	// TelemetryCeilometerIpmiDefaultImage
-	TelemetryCeilometerIpmiDefaultImage = "quay.io/podified-antelope-centos9/openstack-ceilometer-ipmi:current-podified"
-	// NodeExporterDefaultImage -
-	TelemetryNodeExporterDefaultImage = "quay.io/prometheus/node-exporter:v1.5.0"
-)
-
-// SetupAnsibleImageDefaults -
-func SetupAnsibleImageDefaults() {
-	dataplaneAnsibleImageDefaults = dataplanev1.DataplaneAnsibleImageDefaults{
-		Frr:                        util.GetEnvVar("RELATED_IMAGE_OPENSTACK_EDPM_FRR_DEFAULT_IMG", FrrDefaultImage),
-		IscsiD:                     util.GetEnvVar("RELATED_IMAGE_OPENSTACK_EDPM_ISCSID_DEFAULT_IMG", IscsiDDefaultImage),
-		Logrotate:                  util.GetEnvVar("RELATED_IMAGE_OPENSTACK_EDPM_LOGROTATE_CROND_DEFAULT_IMG", LogrotateDefaultImage),
-		NeutronMetadataAgent:       util.GetEnvVar("RELATED_IMAGE_OPENSTACK_EDPM_NEUTRON_METADATA_AGENT_DEFAULT_IMG", NeutronMetadataAgentDefaultImage),
-		NovaCompute:                util.GetEnvVar("RELATED_IMAGE_OPENSTACK_EDPM_NOVA_COMPUTE_DEFAULT_IMG", NovaComputeDefaultImage),
-		OvnControllerAgent:         util.GetEnvVar("RELATED_IMAGE_OPENSTACK_EDPM_OVN_CONTROLLER_AGENT_DEFAULT_IMG", OvnControllerAgentDefaultImage),
-		OvnBgpAgent:                util.GetEnvVar("RELATED_IMAGE_OPENSTACK_EDPM_OVN_BGP_AGENT_IMAGE", OvnBgpAgentDefaultImage),
-		TelemetryCeilometerCompute: util.GetEnvVar("RELATED_IMAGE_OPENSTACK_EDPM_CEILOMETER_COMPUTE_IMAGE", TelemetryCeilometerComputeDefaultImage),
-		TelemetryCeilometerIpmi:    util.GetEnvVar("RELATED_IMAGE_OPENSTACK_EDPM_CEILOMETER_IPMI_IMAGE", TelemetryCeilometerIpmiDefaultImage),
-		TelemetryNodeExporter:      util.GetEnvVar("RELATED_IMAGE_OPENSTACK_EDPM_NODE_EXPORTER_IMAGE", TelemetryNodeExporterDefaultImage),
-	}
+// NodeConfigElements is a struct containing just the elements used for Ansible executions
+type NodeConfigElements struct {
+	BaremetalSetTemplate baremetalv1.OpenStackBaremetalSetSpec
+	NodeTemplate         dataplanev1.NodeTemplate
+	Nodes                map[string]dataplanev1.NodeSection
```

Comment:
Looks like we need to rebase on main. This struct has been removed.

Reply:
Done.